### ISSUE TITLE
New issue: Neural networks shapes

### ISSUE THREAD

**Issue by mcapurso** [sdteffen/dia-additional-shapes#6]
I think it could be useful to define shapes to design neural networks, specifically different types of layers (dense, convolutional, dropout, max pooling ...). And then their parameters (second step). And finally to generate code for Keras/Tensorflow(third step).
